docs/wlc.rst

```diff
@@ -36,61 +36,61 @@ Commands actually indicate which operation should be performed.
 Description
 +++++++++++
 
-Weblate Client is Python library and command line utility to manage Weblate remotely
-using :ref:`api`. The command line utility can be invoked as :command:`wlc` and is
+Weblate Client is a Python library and command-line utility to manage Weblate remotely
+using :ref:`api`. The command-line utility can be invoked as :command:`wlc` and is
 built on :mod:`wlc`.
 
 Global options
 --------------
 
-The program accepts the following global options, which must be entered before subcommand.
+The program accepts the following global options, which must be entered before any subcommand.
 
 .. option:: --format {csv,json,text,html}
```

Comment:
```suggestion
.. option:: --format {CSV,JSON,text,HTML}
```

Reply:
As this is what can be specified on the command line, I'd stick with lower case (and I'm not sure if uppercase will work here, probably not).